examples/pylint/C0121_singleton_comparison.py

```diff
@@ -0,0 +1,14 @@
+"""pylint: singleton comparison
+
+"""
+
+def is_true():
+    """
+    @type obj: int
+    @rtype: bool
+    """
+    temp = 5
+    if temp == True:
+        return False
```

Comment:
No, the condition `if temp is None` does not raise the same warning. Should I leave it as it is then?